docs/DeveloperGuide.md

```diff
@@ -607,44 +607,35 @@ The following activity diagram summarizes what happens when a user wants to add
-| *         | user,  | add payments for each patient                                                                                                  | streamline the payment process and also reducing administrative errors.                      |
-| *         | user,  | close payments for each patient                                                                                                | streamline the payment process and also reducing administrative errors.                      |
-| *         | user,  | view the prescribed medicine for the patient's latest appointment                                                              | dispense the appropriate medicine.                                                           |
-| *         | user,  | view all outstanding bills of each patient to advise                                                                           | let patients are aware of their financial obligations and can plan accordingly.              |
-| *         | user,  | securely transmit patient invoices and billing information to the finance department                                           | streamline the billing process                                                               |
-| *         | user,  | update patient consent forms and privacy policies                                                                              | let patients review and sign electronically                                                  |
-| *         | user,  | customize appointment reminder messages                                                                                        | cater to patients' preferred communication methods (email, SMS, phone call)                  |
+| Priority | As a … | I want to …                                            | So that I can…                                                                                            |
+|----------|--------|--------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| * * *    | user   | create an appointment in the system                    | ensure the the system has a record of this appointment.                                                   |
+| * * *    | user   | delete an appointment from the system                  | ensure the appointment that has been postponed or cancelled will not be reflected in the system.          |
+| * * *    | user   | view the list of appointment in the system             | see the information of the appointment.                                                                   |
+| * * *    | user   | add doctor profile into the system                     | ensure the the system keep tracks of all the doctor working at UHC.                                       |
```

Comment:
i think we should standardize the add/update/delete/view appointment/doctor/patient user stories. Also we not doing UHC anymore

Reply:
Same Feeling but may I clarify, do you mean by the standardizing "so that I can" or all.

Do we need to update the issues in github too?

Reply:
@yanghengtang I think you may want to be involved in this discussion, would appreciate your thoughts. thanks

Reply:
ok will standardise on the google sheet

Reply:
Done, Thanks Derek and Yang Heng